src/gnome_abrt/views.py

```diff
@@ -457,36 +364,45 @@ def option_updated(self, conf, option):
 
 
     def __init__(self, application, sources, controller):
-        Gtk.ApplicationWindow.__init__(self,
-                            title=OopsWindow._TITLE,
-                            application=application)
+        super().__init__(application=application)
 
         if not sources:
             raise ValueError("The source list cannot be empty!")
 
-        self._builder = OopsWindow.OopsGtkBuilder()
-        self._builder.reset_window(self, OopsWindow._TITLE)
+        GObject.Binding.bind_property(
```

Comment:
(idk) is this the regular identation?

Reply:
/me shrugs.

I just moved this bit from another place.